assets/stylesheets/style.scss

```diff
@@ -191,65 +248,69 @@ header {
 //DIAGRAM SECTION
 
 .diagram {
-  position: relative;
   padding-top: 25px;
+  position: relative;
 }
 
-.diagram-details {
+.diagram-detail {
   padding-top: 65px;
-}
 
-.diagram-detail {
-  position: relative;
-  width: 100%;
-  background-color: white;
-  border-radius: 10px;
-  padding: 73px 38px 38px;
-  max-width: 700px;
-  margin: 0 auto;
+  .diagram-detail__card {
+    background-color: white;
+    border-radius: 10px;
+    margin: 0 auto;
+    max-width: 700px;
+    padding: 73px 38px 38px;
+    position: relative;
+    text-align: center;
+    width: 100%;
+
+    .diagram-detail__card__content {
+      @extend %text-small;
+      line-height: 1.375
+    }
+  }
+
+  &.-last {
+    margin-bottom: 40px
+  }
 }
 
 .icon {
-  position: absolute;
   border-radius: 100px;
-  right: 0;
   left: 0;
   margin: 0 auto;
-}
-.icon.-borders{
-  width: 88px;
-  height: 88px;
-  top: -33.5px;
-  border: 10px solid rgba(255, 255, 255, 0.57);
-}
+  position: absolute;
+  right: 0;
 
+  &.-borders {
+    border: 10px solid rgba(255, 255, 255, 0.57);
```

Comment:
With Scss you can do `border: 10px solid rgba(#fff, 0.57);` and the RGB values will get converted.